.github/workflows/release.yaml

```
@@ -0,0 +1,36 @@
+name: Release a new version
+on:
+  push:
+    tags:
+      - "v*"
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Set up Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.19
```

Are you up to going with 1.20 by #109?

```suggestion
          go-version: '1.20'
```